src/scripts/components/main/language-selector/language-selector.tsx

```diff
@@ -21,6 +21,14 @@ const LanguageSelector: FunctionComponent<Props> = ({className = ''}) => {
   const selectedLanguage = useSelector(languageSelector);
   const dispatch = useDispatch();
 
+  const setLanguage = (language: Language) => {
+    localStorage.setItem('language', language);
```

Comment:
It would be cool if the saving to localStorage and the dispatch would be combined so that we don't have to handle the saving manually in multiple places. We could just always save the language to local storage when dispatching the `setLanguageAction` action in the `actions/set-language.ts` file